### PR TITLE
fix(sg): limit the creation of negative ICMP type/code for IPv6

### PR DESCRIPTION
**Pull request by tamazlykar**

### CODE REVIEW

File: src/app/security-group/sg-rules/sg-rules.component.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
                 <input
                   matInput
                   maxValue="255"
-                  [minValue]="cidrIpVersion === IPVersion.ipv6 ? 0 : -1"
+                  minValue="0"
```
Comment (qqcky) [inline src/app/security-group/sg-rules/sg-rules.component.html]: It should be possible to use ICMP type -1 for IPv4 rules. The same for ICMP codes.

File: src/app/security-group/sg-rules/sg-rules.component.html [hunk 2]
```diff
@@ -109,7 +109,7 @@
                   matInput
                   name="icmpCode"
                   maxValue="255"
-                  [minValue]="cidrIpVersion === IPVersion.ipv6 ? 0 : -1"
+                  minValue="0"
```
Comment (qqcky) [inline src/app/security-group/sg-rules/sg-rules.component.html]: It should be possible to use ICMP code -1 for IPv4 rules. The same for ICMP types.

File: src/app/security-group/sg-rules/sg-rules.component.html (hunk 1 above)
Comment (tamazlykar) [reply src/app/security-group/sg-rules/sg-rules.component.html]: Please provide link that ICMP Type can have "-1" value. I can't find that info on the internet.
https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml

File: src/app/security-group/sg-rules/sg-rules.component.html (hunk 2 above)
Comment (tamazlykar) [reply src/app/security-group/sg-rules/sg-rules.component.html]: Please provide link that ICMP Type can have "-1" value. I can't find that info on the internet.
https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml

**Comment by tamazlykar** [comment]
@qqcky Okay, @bwsw explained that Cloudstack interpret -1 any value
I'll add comments for this in code

**Pull request merged**